requirements/project-requirements.txt

```diff
@@ -110,4 +110,5 @@ DeepDiff==8.2.0
 # phishing form compiler module
 lxml==5.3.0
 Faker==36.1.0
-beautifulsoup4==4.13.4
\ No newline at end of file
+beautifulsoup4==4.13.4
+jbxapi==3.23.0
```

Comment:
add it upper in the list cause it risks to be confused with the previous comment